### ISSUE TITLE
 Unable to send message in tags page

### ISSUE THREAD

**Issue by JumpingY** [justoverclockl/flachat#1]
Flarum1.1.0
PHP7.4.3
MySQL5.7.26
Unadle to send message in tags page，but I can send message in other categories page

**Comment by justoverclockl** [comment]
Yep that’s because Actually the extender is Indexpage, I will fix this soon
